boards/aithinker/ai_m62_12f_kit/ai_m62_12f.dtsi

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2025 MASSDRIVER EI (massdriver.space)
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
```

Comment:
Can eventually be removed now that it's a `.dtsi` (it is already in the `.dts`).